shared/test/textChatView_test.js

```diff
@@ -203,26 +203,6 @@ describe("loop.shared.views.TextChatView", function() {
           .to.eql(2);
     });
 
-    it("should show timestamps if they are 1 minute apart (SENT)", function() {
```

Comment:
I was operating under the theory "that's what revision control history is for", but ok.